Extension/src/common.ts

```diff
@@ -331,6 +331,17 @@ export function resolveVariables(input: string | undefined, additionalEnvironmen
                     }
                     if (newValue === undefined) {
                         newValue = process.env[name];
+                        if (!newValue) {
```

Comment:
These special named things are normally added to the "additionalEnvironment."  Instead of adding special code for them here, I would recommend searching the codebased for callers of this function and add them to the "additionalEnvironment" instead.